lib/home/Welcome.jsx

```diff
@@ -29,7 +29,7 @@ const Welcome = ({ playerName }) => {
             <Text style={styles.title}>Tipsy</Text>
             <Text style={styles.text}>Welcome {playerName}</Text>
             <AdaptiveButton
-                onPress={handleNewGame}
+                onPress={() => handleNewGame(false)}
```

Comment:
If `quickGame` is `false` by default in `handleNewGame`, you don't need to pass it right? Not important though

Reply:
It didn't work when I did it... Don't understand why, and no time to investigate :)